docs/managing-e2e-tests.md

```diff
@@ -23,106 +24,140 @@ It contains a list of jobs such as:
   ...
 ```
 
-In this case, `ci-kubernetes-e2e-kubeadm-kind-master` is a test job that runs every 2 hours and it also has a set of parameters defined for it. This document will only cover some of them.
+In this case, `ci-kubernetes-e2e-kubeadm-kind-master` is a test job that runs every 2 hours and it also
+has a set of other parameters defined for it. Such test jobs use an image that will run as a container inside
+a Pod of the Kubernetes [Prow](https://github.com/kubernetes/test-infra/tree/master/prow) cluster.
 
-Important to note here is that testgrid jobs have a docker `image` that contains all the tools needed to run tests. Among them, one of the most relevant ones, is the tool that will be used for the deployment of the cluster.
+For this example job the deployment tool is called [kind](https://github.com/kubernetes-sigs/kind) (with integration
```

Comment:
I'm not sure all those detail are useful/easy to understand for a broader audience.
what about leaving only the high level summary?

Reply:
removed the kind.go integration links.